diff -ruN a/extras/Build/CMake/JUCEUtils.cmake b/extras/Build/CMake/JUCEUtils.cmake
--- a/extras/Build/CMake/JUCEUtils.cmake	2021-09-20 14:26:04.000000000 +0200
+++ b/extras/Build/CMake/JUCEUtils.cmake	2021-09-25 19:28:31.202649065 +0200
@@ -87,6 +87,7 @@ set_property(GLOBAL PROPERTY JUCE_COPY_PLUGIN_AFTER_BUILD FALSE)
 if((CMAKE_SYSTEM_NAME STREQUAL "Linux") OR (CMAKE_SYSTEM_NAME MATCHES ".*BSD"))
     _juce_create_pkgconfig_target(JUCE_CURL_LINUX_DEPS libcurl)
     _juce_create_pkgconfig_target(JUCE_BROWSER_LINUX_DEPS webkit2gtk-4.0 gtk+-x11-3.0)
+    _juce_create_pkgconfig_target(JUCE_IMG_LINUX_DEPS libjpeg libpng)
 endif()
 
 # We set up default/fallback copy dirs here. If you need different copy dirs, use
@@ -168,6 +169,8 @@ function(_juce_link_optional_libraries target)
         if(needs_browser)
             target_link_libraries(${target} PRIVATE juce::pkgconfig_JUCE_BROWSER_LINUX_DEPS)
         endif()
+
+        target_link_libraries(${target} PRIVATE juce::pkgconfig_JUCE_IMG_LINUX_DEPS)
     elseif(APPLE)
         get_target_property(needs_storekit ${target} JUCE_NEEDS_STORE_KIT)
 
